modules/effects/arrow-effects-kotlinx-coroutines/src/main/kotlin/arrow/effects/DeferredK.kt

```diff
@@ -548,9 +549,32 @@ sealed class DeferredK<A>(
           result
         }
       }
+
+    fun <A, B> racePair(ctx: CoroutineContext,
+                        lh: DeferredKOf<A>,
+                        rh: DeferredKOf<B>): DeferredK<Either<Tuple2<A, Fiber<ForDeferredK, B>>, Tuple2<Fiber<ForDeferredK, A>, B>>> =
+      lh.startF(ctx).flatMap { fiberA ->
+        rh.startF(ctx).flatMap { fiberB ->
+          DeferredK.async<Either<Tuple2<A, Fiber<ForDeferredK, B>>, Tuple2<Fiber<ForDeferredK, A>, B>>> { cb ->
+            fiberA.join.unsafeRunAsync { eith ->
```

Comment:
What happens with this pair on cancellation of the parent `DeferredK`? Do we have a test for that? They don't seem connected by sharing a parent `Job()` for example.

Reply:
This is removed and postponed to become a separate ticket.